tests/autoyast/wicked.pm

```diff
@@ -27,7 +27,7 @@ use testapi;
 
 sub run {
     # https://en.opensuse.org/openSUSE:Bugreport_wicked
-    enter_cmd "systemctl status wickedd.service";
+    enter_cmd "systemctl status --no-pager wickedd.service";
```

Comment:
For me this looks like a workaround while not found the root cause. 
We'd better to know why wicked will not return, maybe this is a bug, or something wrong for the network setting in the profile.

Reply:
I just fixed the ticket. For the root cause, we need check later.

Reply:
same that it does `lib/Utils/Systemd.pm` in `sub systemcl` we should use that functions for most of the cases btw.
if the status pipe out to page is normal that the automation cannot work.
now...why doesn't ellipsize lines like before, it might be that the defaults have changed ...
we should resolve that question before merging IMO.

Reply:
Updated, use 'systemctl' function, It can work. 
Thanks for your suggestion.